src/components/notifications/ConfigurationTab.tsx

```diff
@@ -202,6 +239,86 @@ export class ConfigurationTab extends Component<{}, ConfigurationTabState> {
+                            <td className="slack-config-table__action"></td>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr className="mb-8">
+                            {this.state.webhookConfigurationList.map((webhookConfig) => {
+                                return (
+                                    <td key={webhookConfig.id} className="configuration-tab__table-row" data-testid={`webhook-container-${webhookConfig.name}`}>
+                                        <div className="slack-config-table__name dc__truncate-text" data-testid={`webhook-config-name-${webhookConfig.name}`}>
+                                            {webhookConfig.name}
+                                        </div>
+                                        <div className="slack-config-table__webhook dc__truncate-text" data-testid={`webhook-url-${webhookConfig.webhookUrl}`}>
+                                            {webhookConfig.webhookUrl}
+                                        </div>
+                                        <div className="slack-config-table__action">
+                                            <Tippy className="default-tt" arrow={false} placement="top" content="Edit">
+                                                <button
+                                                    type="button"
+                                                    className="dc__transparent dc__align-right mr-16"
+                                                    onClick={(event) => {
+                                                        this.setState({
+                                                            showWebhookConfigModal: true,
+                                                            webhookConfigId: webhookConfig.id,
+                                                        })
+                                                    }}
+                                                    data-testid="webhook-configure-edit-button"
+                                                >
+                                                    <Edit className="icon-dim-20" />
+                                                </button>
+                                            </Tippy>
+                                            <Tippy
+                                                className="default-tt"
+                                                arrow={false}
+                                                placement="top"
+                                                content="Delete"
+                                            >
+                                                <button
+                                                    type="button"
+                                                    className="dc__transparent dc__align-right"
+                                                    onClick={() => {
```

Comment:
add id to data-webhookConfig.id & then use it in function & remove this inline

Reply:
resolved